ios/RCTWeChat.xcodeproj/project.pbxproj

```diff
@@ -219,8 +221,8 @@
 			buildSettings = {
 				HEADER_SEARCH_PATHS = (
 					"$(inherited)",
-					"$(SRCROOT)/../../react-native/React/**",
-					"$(SRCROOT)/../../react-native/Libraries/Image/**",
+					"$(SRCROOT)/../Example/node_modules/react-native/React/**",
```

Comment:
Please do not change the Header Search Path.
Old path like "$(SRCROOT)/../../react-native/React/**" is suitable in production environment.
New path like "$(SRCROOT)/../Example/node_modules/react-native/React/**" is only suitable in development environment. 

Example/node_modules directory will never exists in production environment. but the node_modules will beside within project root.